engine-api/src/main/java/io/nosqlbench/engine/api/scenarios/NBCLIScenarioParser.java

```diff
@@ -230,6 +230,12 @@ private static LinkedHashMap<String, CmdArg> parseStep(String cmd) {
         String[] namedStepPieces = cmd.split(" ");
         for (String commandFragment : namedStepPieces) {
             Matcher matcher = WordAndMaybeAssignment.matcher(commandFragment);
+
+            if (commandFragment.equalsIgnoreCase("")) {
+                logger.debug("Being kind, command fragment was empty.");
```

Comment:
Can you elaborate on this message? I'm not sure it conveys enough context.

Reply:
Updated.